tutorials/ui/gui_containers.rst

```diff
@@ -34,24 +34,30 @@ Example of *HBoxContainer* resizing children buttons.
 
 The real strength of containers is that they can be nested (as nodes), allowing the creation of very complex layouts that resize effortlessly.
 
-Size flags
+Sizing options
 ----------
```

Comment:
The underline needs to be updated to match the new title length.

Reply:
Done in commit [253f4ba](https://github.com/godotengine/godot-docs/pull/6636/commits/253f4ba9ed6daa057033191802890946e8cfb368)